docs/releases/1.9.txt

```diff
@@ -319,6 +319,9 @@ Requests and Responses
 * The default 40x error views now accept a second positional parameter, the
   exception that triggered the view.
 
+* Exceptions raised by the :meth:`render() <django.template.response>` method
+  are now correctly passed to the middlewares ``process_exception`` methods.
```

Comment:
"the render method of template responses" (chop link - not sure it adds clarifty?)
chop "correctly"?
passed to the process_exception() method of each middleware.